File: Segundo corte/Taller bucles/Punto1.py
Ovejas = [ "Oveja1", "Oveja2", "Oveja3", "Oveja4", "Oveja5"]

Contador_oveja = 1
while Contador_oveja < 5 :
    Nombre_oveja = Ovejas[Contador_oveja - 1] 
    print (f"La oveja número: {Contador_oveja} llamada {Nombre_oveja} se encuentra aquí")
    Contador_oveja += 1

print("Todas las ovejas han sido contadas y no falta ninguna")

File: Segundo corte/Taller bucles/Punto1.cpp
#include <iostream>
using namespace std;

int main(){

string Lista_Ovejas[5] = { "Oveja1" , "Oveja2" , "Oveja3" , "Oveja4" , "Oveja5" };

int Contador_ovejas = 1;

while (Contador_ovejas < 6){

    cout << "La oveja número: " << Contador_ovejas << " llamada: " << Lista_Ovejas[Contador_ovejas - 1] << " se encuentra aquí" << endl;
    Contador_ovejas ++;
}
cout << "Todas las ovejas se encuentran aquí" ;

}
